src/devdb/helpers.py

```diff
@@ -1,3 +1,4 @@
+from re import M
```

Comment:
what is this

Reply:
it's absolutely infuriating that my vscode being just adding random imports to my scripts now.

Reply:
think I found a way to turn this feature off

Reply:
okay also googled and turned the feature off. Hopefully no more this in the future.